docs/userguides/quickstart.md

```diff
@@ -201,3 +240,11 @@ c.viewThis()  # Make Web3 calls
 c.doThat(sender=a)  # Make Web3 transactions
 assert c.MyEvent[-1].caller == a  # Search through Web3 events
 ```
+
+## Deployments
```

Comment:
```suggestion
## Scripts
```

And then maybe talk about how `deploy` scripts are a common type of script to make.

Reply:
I like that idea